File: 윤석주/day4/주식가격.py
def solution(prices):
    answer = []
    for i in range(len(prices)-1):
        start = 0
        for j in range(i+1,len(prices)):
            start+=1
            if prices[j] < prices[i]:
                break
        answer.append(start)
    answer.append(0)
    return answer

File: 박재현/이전/주식가격.cpp
#include <vector>
#include <stack>
#include <algorithm>
using namespace std;

vector<int> solution(vector<int> prices)
{
    int size = prices.size();
    vector<int> answer(size);
    stack<int> st;

    for (int i = 0; i < size; i++)
    {
        while (!st.empty() and prices[st.top()] > prices[i])
        {
            answer[st.top()] = i - st.top();
            st.pop();
        }

        st.push(i);
    }
    while (!st.empty())
    {
        answer[st.top()] = size - 1 - st.top();
        st.pop();
    }
    return answer;
}